src/Employer/Employer.Web/Controllers/ApplicationReviewController.cs

```diff
@@ -28,15 +30,43 @@ public async Task<IActionResult> ApplicationReview(ApplicationReviewRouteModel r
         [HttpPost("", Name = RouteNames.ApplicationReview_Post)]
         public async Task<IActionResult> ApplicationReview(ApplicationReviewEditModel m)
         {
-            if (ModelState.IsValid == false)
+            if (!ModelState.IsValid)
             {
                 var vm = await _orchestrator.GetApplicationReviewViewModelAsync(m);
                 return View(vm);
+            }            
+            var applicationStatusConfirmationViewModel = await _orchestrator.GetApplicationStatusConfirmationViewModelAsync(m);
+            return View("ApplicationStatusConfirmation", applicationStatusConfirmationViewModel);
+        }
+
+       [HttpPost("status", Name = RouteNames.ApplicationReviewConfirmation_Post)]
+        public async Task<IActionResult> ApplicationStatusConfirmation(ApplicationReviewStatusConfirmationEditModel applicationReviewStatusConfirmationEditModel)
+        {
+            if (!ModelState.IsValid)
+            {
+                var vm = await _orchestrator.GetApplicationStatusConfirmationViewModelAsync(applicationReviewStatusConfirmationEditModel);
+                return View(vm);
             }
 
-            await _orchestrator.PostApplicationReviewEditModelAsync(m, User.ToVacancyUser());
+            if (CanNotifyApplicant(applicationReviewStatusConfirmationEditModel))
+            {
+                await _orchestrator.PostApplicationReviewEditModelAsync(applicationReviewStatusConfirmationEditModel, User.ToVacancyUser());
+                var routeModel = new ApplicationReviewRouteModel {
+                    ApplicationReviewId = applicationReviewStatusConfirmationEditModel.ApplicationReviewId,
+                    VacancyId = applicationReviewStatusConfirmationEditModel.VacancyId,
+                    EmployerAccountId = applicationReviewStatusConfirmationEditModel.EmployerAccountId
+                };
+                var vm = await _orchestrator.GetApplicationReviewViewModelAsync(routeModel);
```

Comment:
(line 54-59) This feels a bit hacky. Can the PostApplicationReviewEditModelAsync return the name of the applicant/candidate? Then you won't need to do any of this.